src/lib/Behat/PageObject/Page.php

```diff
@@ -101,4 +101,14 @@ public function getPageTitle(): string
     {
         return $this->context->findElement($this->pageTitleLocator)->getText();
     }
+
+    /**
+     * Gets the route URL text displayed in AdminUI page.
+     *
+     * @return string
+     */
+    public function getPageRoute(): string
```

Comment:
```suggestion
    protected function getCurrentRoute(): string
```

1) `protected`, because it should not be used in Context classes
2) getCurrentRoute, because current naming suggests that it returns the `route` property
3) verifyRoute should use this function (this is a method extracted from verifyRoute)